### PR TITLE
feat(MWPW-125698): lana logging

### PR DESCRIPTION
**Pull request by sheridansunier**
Integrates CaaS with Lana and adds a first pass of lana logs.

The primary focus of these logs is to add logging around our collection endpoint calls.  I also added some logging around try/catch blocks within reason.  After we have time seeing the implicit logging we get we can choose what kind of coverage and logs we want to add.

### CODE REVIEW

File: react/src/js/components/.DS_Store
Comment (cmiqueo) [inline react/src/js/components/.DS_Store:1]: WE should `.DS_Store` to the .gitignore list

**Review by cmiqueo — APPROVED** [review]

File: react/src/js/components/.DS_Store
Comment (sheridansunier) [reply react/src/js/components/.DS_Store:1]: ok silly question... but once I have updated .gitignore is there a good way to remove those .DS_Store files from this PR?

File: react/src/js/components/Consonant/Helpers/lana.js [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+            window.lana.log(e.reason || e.error || e.message, { errorType: 'i' });
+        }
+    };
+
+    window.lana = {
+        log: async (...args) => {
+            window.removeEventListener('error', lanaError);
+            window.removeEventListener('unhandledrejection', lanaError);
+            // eslint-disable-next-line import/no-unresolved, import/extensions
+            await fetch('https://milo.adobe.com/libs/utils/lana.js');
```
Comment (jedjedjedM) [inline react/src/js/components/Consonant/Helpers/lana.js:15]: This should load from `www.adobe.com/libs/utils/lana.js` instead of milo.adobe.com

Comment (sheridansunier) [reply react/src/js/components/Consonant/Helpers/lana.js]: ok updated

**Pull request merged**